src/org/labkey/test/util/Crawler.java

```diff
@@ -1018,7 +1021,23 @@ else if (downloadDir.isDirectory()) // Don't check for download if dir doesn't e
 
                     if (depth >= 0 && !_terminalActions.contains(actionId)) // Negative depth indicates a one-off check
                     {
-                        List<String> linkAddresses = _test.getLinkAddresses();
+                        List<Pair<String, Map<String, String>>> linksWithAttributes = _test.getLinkAddresses();
+                        for (Pair<String, Map<String, String>> linkWithAttributes : linksWithAttributes)
+                        {
+                            String href = linkWithAttributes.getLeft();
+                            if (href.contains("://") && !href.startsWith(WebTestHelper.getBaseURL())) // Remote URL
+                            {
+                                Map<String, String> attributes = linkWithAttributes.getRight();
+                                String target = StringUtils.trimToEmpty(attributes.get("target"));
+                                List<String> rel = Arrays.asList(StringUtils.trimToEmpty(attributes.get("rel")).split(" +"));
+                                if (target.equals("_blank"))
+                                {
+                                    MatcherAssert.assertThat("Bad 'rel' attribute for link to " + href,
```

Comment:
Will the logging prior to this assert make it clear which page contained the bad link?